src/styles/blocks/header.scss

```diff
@@ -0,0 +1,73 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+
+  padding: 0 50px;
+}
+
+.logo {
```

Comment:
The block name should follow the BEM naming conventions. Instead of just '.logo', it could be named '.header__logo' to indicate that it is a part of the 'header' block.